PetrolEngine.cpp

```diff
@@ -2,19 +2,20 @@
 #include <iostream>
 
 PetrolEngine::PetrolEngine(int power, float capacity, int gears)
-    : power_(power)
-    , capacity_(capacity)
-    , gears_(gears)
-    , currentGear_(0)
-{
-    std::cout << __FUNCTION__ << std::endl;
+    : Engine(power), capacity_(capacity), gears_(gears), currentGear_(0) {
 }
 
-void PetrolEngine::changeGear(int gear)
-{
+void PetrolEngine::changeGear(int gear) {
     // TODO: Add checking if gear is between -1 and gears_
     // -1 is for REAR
     // 0 is for NEUTRAL
+
```

Comment:
Why do you keep TODO comment after it's done.

Reply:
forgot about it